cage.c

```diff
@@ -324,7 +324,7 @@ main(int argc, char *argv[])
 	wl_list_init(&server.views);
 	wl_list_init(&server.outputs);
 
-	server.output_layout = wlr_output_layout_create();
+	server.output_layout = wlr_output_layout_create(server.wl_display);
```

Comment:
Is this still needed?
```
wlr_output_layout_destroy(server.output_layout);
```
https://github.com/cage-kiosk/cage/blob/e61e1d3dcf70c77155e7c1d9f6b2736c741152a9/cage.c#L593C1-L593C50

Reply:
Indeed, it's now destroyed automatically. Removed!